Docs/SPE 260: update and reorganize telemetry metric references (#21965)
hashicorp/vault: website/content/partials/telemetry-metrics/vault/raft/get.mdx

### vault.raft.get ((#vault-raft-get))

Metric type | Value | Description
----------- | ----- | -----------

> summary     | ms    | Time required to retrieve a file from underlying storage

Reviewer: ```suggestion
summary     | ms    | Time required to retrieve an entry from underlying storage
```

> summary     | ms    | Time required to retrieve a retrieve a file from underlying storage

Reviewer 2: ```suggestion
summary     | ms    | Time required to retrieve a file from underlying storage
```
